packages/tenant-process/test/addCertifiedAttribute.test.ts

```diff
@@ -77,7 +77,7 @@ describe("addCertifiedAttribute", async () => {
     await addOneTenant(targetTenant);
     await addOneAttribute(attribute);
     await addOneTenant(requesterTenant);
-    const returnedTenant = await tenantService.addCertifiedAttribute(
+    const { data: returnedTenant } = await tenantService.addCertifiedAttribute(
```

Comment:
As done for attributes/delegations, I would update this test to test the entire result object including metadata

See https://github.com/pagopa/interop-be-monorepo/pull/1849/files#diff-2ab69d62d3cac341a38dac792805f097aee52ca413a8806d8b976398bf47d1afR82